docker/drama-free-django/_test.sh

```diff
@@ -45,6 +45,8 @@ export DJANGO_SETTINGS_MODULE=cfgov.settings.production
 # commands do nothing.
 mkdir -p static.in/0/fonts
 
+export DJANGO_STATICFILES_IN=/tmp/current/static.in
```

Comment:
Is this a legacy addition that can now be removed since we're using local static.in?